### PR TITLE
swapping the content of the contibuting doc and the readme

### PR DESCRIPTION
**Pull request by sbvegan**
**Description**

A clear and concise description of the features you're adding in this pull request.

**Tests**

Please describe any tests you've added. If you've added no tests, or left important behavior untested, please explain why not.

**Additional context**

Add any other context about the problem you're solving.

**Metadata**

- Fixes #[Link to Issue]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,108 +18,172 @@ It is meant to be used when building transactions that upgrade the implementatio
 
 If you would like to contribute a new chain or superchain, please see our [contributing guide](./CONTRIBUTING.md).
```
Comment (tessr) [inline README.md:19]: Is this callout still necessary, given that the instructions are now directly in this readme?

Comment (sbvegan) [reply README.md:19]: nope, just missed that, thanks!

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # superchain-registry
 
-> ⚠️ This repository is a **work in progress**.  At a later date, it will be proposed to, and must be approved by, Optimism Governance.  Until that time, the configuration described here is subject to change.
+> :warning: This repository is a **work in progress**.  At a later date, it will be proposed to, and must be approved by, Optimism Governance.  Until that time, the configuration described here is subject to change.
+> 
+> :no_entry: We're currently not accepting additional chains to be added to the registry. We'll reopen this process when the repository is ready.
```
Comment (tessr) [inline README.md:5]:
```suggestion
> :no_entry: We're making some changes to this repository and we've paused adding new chains for now. We'll reopen this process once the repository is ready. The Superchain itself, of course, remains open for business.
```

Comment (tessr) [reply README.md:5]: (just so people don't think that we're not adding new chains to the Superchain itself)

**Review by mds1 — APPROVED** [review]
LGTM! Just one nit, feel free to ignore it

**Review by geoknee — CHANGES_REQUESTED** [review]
Thanks @sbvegan, could you review these extra changes #248 and merge them into this PR if happy?

**Review by geoknee — APPROVED** [review]

**Pull request merged**